src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -33,291 +34,295 @@
-              <Button mnemonicParsing="false" onAction="#buttonAddCandidateClicked" text="Add"/>
-              <Region prefHeight="25.0" prefWidth="39.0"/>
-              <Button mnemonicParsing="false" onAction="#buttonDeleteCandidateClicked"
-                text="Delete Selected"/>
-              <padding>
-                <Insets bottom="4.0" left="4.0" right="4.0" top="4.0"/>
-              </padding>
-            </HBox>
-            <Label
-              text="Note: double-click existing entries below to edit and press Enter to save changes.">
-              <padding>
-                <Insets bottom="2.0" left="6.0"/>
-              </padding>
-            </Label>
-            <TableView fx:id="tableViewCandidates">
-              <columns>
-                <TableColumn minWidth="20.0" onEditCommit="#changeCandidateName"
-                  prefWidth="230.0" text="Name" fx:id="tableColumnCandidateName"/>
-                <TableColumn minWidth="20.0" onEditCommit="#changeCandidateCode"
-                  prefWidth="100.0" text="Code" fx:id="tableColumnCandidateCode"/>
-                <TableColumn minWidth="20.0" prefWidth="100.0" text="Excluded"
-                  fx:id="tableColumnCandidateExcluded"/>
-              </columns>
-            </TableView>
-            <padding>
-              <Insets top="12.0"/>
-            </padding>
-          </VBox>
-        </Tab>
-        <Tab text="Rules">
-          <HBox>
-            <VBox spacing="8.0">
-              <Label text="Tiebreak Mode *"/>
-              <ChoiceBox prefWidth="220.0" fx:id="choiceTiebreakMode"/>
-              <Label text="Overvote Rule *"/>
-              <ChoiceBox prefWidth="220.0" fx:id="choiceOvervoteRule"/>
-              <Label text="Number of Winners *"/>
-              <TextField maxWidth="220.0" promptText="Number of Winners"
-                fx:id="textFieldNumberOfWinners"/>
-              <Label text="Decimal Places for Vote Arithmetic *"/>
-              <TextField maxWidth="220.0" promptText="Decimal Places for Vote Arithmetic"
-                fx:id="textFieldDecimalPlacesForVoteArithmetic"/>
-              <Label text="Minimum Vote Threshold *"/>
-              <TextField maxWidth="220.0" promptText="Minimum Vote Threshold"
-                fx:id="textFieldMinimumVoteThreshold"/>
-              <Label text="Max Skipped Ranks Allowed"/>
-              <TextField maxWidth="220.0" promptText="Max Skipped Ranks Allowed"
-                fx:id="textFieldMaxSkippedRanksAllowed"/>
-              <Label text="Max Rankings Allowed"/>
-              <TextField maxWidth="220.0" promptText="Max Rankings Allowed"
-                fx:id="textFieldMaxRankingsAllowed"/>
-              <padding>
-                <Insets bottom="12.0" left="12.0" right="12.0" top="12.0"/>
-              </padding>
-            </VBox>
+  <BorderPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity"
```

Comment:
the maxes shouldn't be positive infinity?

Reply:
Weird, huh? It's a standard value created by SceneBuilder, and I see it in all sorts of examples online. Assuming it's fine, since it seems to be.

Regarding your main comments: yes, only scrollbars when necessary. Filed #303 for your other question.